src/Component/Header/Header.tsx

```diff
@@ -1,10 +1,17 @@
 import React from "react";
-
-export const Header = () => {
+import { Link } from "react-router-dom"
+export const Header: React.FC = () => {
   return (
-    <nav className="navbar">
-      <h1 className="banner">POKEMON CARDS!</h1>
-      <h3>Browse cards and build your deck</h3>
-    </nav>
+    <header>
+      <Link to="/">
+        <img src="" alt="Logo" />
```

Comment:
Love that we'll have an image in the header to route back to the home page instead of a home button - I'll get that in ASAP